docs/tutorial/tutorial.md

```diff
@@ -1029,7 +1029,7 @@ Add a method called `jumpTo` to the Game class:
   jumpTo(step) {
     this.setState({
       stepNumber: step,
-      xIsNext: (step % 2) ? false : true,
+      xIsNext: !(step % 2),
```

Comment:
I would rewrite this as `xIsNext: (step % 2) === 0`. That's the intended logic right?